src/main/java/nl/smallproject/www/techiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,48 @@
+package nl.smallproject.www.techiteasy.controllers;
+
+import nl.smallproject.www.techiteasy.models.Television;
+import nl.smallproject.www.techiteasy.services.TelevisionService;
+import org.apache.coyote.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/televisions")
+public class TelevisionController {
+
+    private final TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<Television>> getAllTelevisionsByBrand(@RequestParam(required = false) String brand) {
+        List<Television> televisions = televisionService.getAllTelevisionsByBrand(brand);
+        return ResponseEntity.ok(televisions);
+    }
+
+    @PostMapping
+    public ResponseEntity<Television> createTelevision(@RequestBody Television television) {
+        Television savedTelevision = televisionService.saveTelevision(television);
+        return ResponseEntity.status(HttpStatus.CREATED).body(savedTelevision);
+    }
+
+    @RequestMapping(value ="{id}", method = RequestMethod.PUT)
```

Comment:
Dit is een valide methode om een PutMapping te maken en het is goed dat je er mee experimenteerd. De `@`PutMapping methode is eigenlijk een shorthand voor dit. Maar als je hier boven wel de shorthand gebruikt voor de Post en hier niet, dan wordt het rommelig. Dus kies voor je eindopdracht strakt 1 lijn, shorthand of geen shorthand? Uniformiteit is belangrijk